index.js

```diff
@@ -0,0 +1,9 @@
+var express = require("express");
+
+var server = express()
+server.get("/", function(req, resp) {
+	resp.send("happy time");
+})
+server.listen(8080, function() {
+	console.log("server started");
```

Comment:
The like to put the port the server is listening on as well in the console message. "server started on port 8080"